CMakeLists.txt

```diff
@@ -78,7 +79,6 @@ else()
   add_cxx_compiler_flag(-pedantic-errors)
   add_cxx_compiler_flag(-Wshorten-64-to-32)
   add_cxx_compiler_flag(-Wfloat-equal)
-  add_cxx_compiler_flag(-Wzero-as-null-pointer-constant)
```

Comment:
I removed this warning because GCC generates it when compiling w/ libc++ even though the actual warning originates from a system header.